src/components/button/button.ts

```diff
@@ -156,8 +163,11 @@ class BXButton extends FocusMixin(LitElement) {
       [`${prefix}--btn--${kind}`]: kind,
       [`${prefix}--btn--disabled`]: disabled,
       [`${prefix}--btn--icon-only`]: hasIcon && !hasMainContent,
-      [`${prefix}--btn--${size}`]: size,
+      [`${prefix}--btn--sm`]: size === 'sm' && !isExpressive,
+      [`${prefix}--btn--xl`]: size === 'xl',
+      [`${prefix}--btn--field`]: size === 'field' && !isExpressive,
       [`${prefix}-ce--btn--has-icon`]: hasIcon,
+      [`${prefix}--btn--expressive`]: isExpressive,
```

Comment:
Do we use this class name to style at all? Seems like we only use the attribute to style

Reply:
![Screen Shot 2021-06-15 at 10 04 45 AM](https://user-images.githubusercontent.com/20210594/122094524-1fc02380-cdc1-11eb-92d5-30057c943bec.png)

yeah the class name is being styled from carbon core

Reply:
ohh ok gotcha, cool!